core/network-libp2p/src/custom_proto/handler.rs

```diff
@@ -350,7 +385,6 @@ where
 	fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
 		f.debug_struct("CustomProtosHandler")
 			.field("protocols", &self.protocols.len())
-			.field("state", &self.state)
```

Comment:
Couldn't this be useful for debugging?

Reply:
This `fmt::Debug` is basically useless because we don't have access to that struct after initialization anymore as we send it to a background task.